tests/change-password.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+import { LoginPage  as UILoginPage } from '../src/pages/ui';
+import { SettingsPage } from '../src/pages/ui';
+let password1 = process.env.HOBBYFARM_ADMIN_UI_PWD;
+let password2 = process.env.HOBBYFARM_ADMIN_UI_PWD_TEST;
```

Comment:
HOBBYFARM_ADMIN_UI_PWD_TEST must be added to .env.example (so people knows they have to set it) and to .github/workflows/ci.yaml and .github/workflows/nightly.yaml as env parameters with others (so it can be taken during the pipeline execution)